### PR TITLE
site/docs: Update AWS NLB Guide with correct annotations

### PR DESCRIPTION
**Pull request by yankay**
Update AWS NLB Guide with correct annotations. Follow the  [AWS EKS Doc](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)

Fix the Issue: https://github.com/projectcontour/contour/issues/4346 .

Signed-off-by: Kay Yan [kay.ay@daocloud.io](mailto:kay.ay@daocloud.io)

### CODE REVIEW

**Comment by sunjayBhatia** [comment]
Thanks for the PR @yankay

Could you also add a release note for this change following: https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md#commit-message-and-pr-guidelines

Though this is a small change, would be good to call out that we are updating the guidance on this, we can try to get an AWS account to test this out with as well

**Comment by yankay** [comment]
HI, @sunjayBhatia . The changlog has been added. And I test it in the AWS, and fix the port from 80  to 443

File: changelogs/unreleased/4347-yankay-docs.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Update AWS NLB Guide with correct annotations.
```
Comment (youngnick) [inline changelogs/unreleased/4347-yankay-docs.md:1]: Could you make this a bit more descriptive? The idea is that someone who reads this in the changelog will understand what the change has done.

Maybe something like:
```
The AWS deployment guide has been updated with the current Service annotations, and should now work correctly with the given YAMLs.
```

**Review by youngnick — COMMENTED** [review]
This is looking great, thanks @yankay! Just one small thing with the changelog item, and then this will LGTM.

File: changelogs/unreleased/4347-yankay-docs.md (hunk 1 above)
Comment (yankay) [reply changelogs/unreleased/4347-yankay-docs.md]: Hi @youngnick , thanks for your comments, the changelog has been fixed.

**Review by youngnick — APPROVED** [review]
Awesome, thanks.

**Review by OrlinVasilev — APPROVED** [review]

**Review by sunjayBhatia — APPROVED** [review]

**Comment by yankay** [comment]
Thanks

**Pull request merged**